sdk/src/main/res/values/strings.xml

```diff
@@ -86,4 +86,6 @@
     <string name="payment_method_internet_banking_ktb_title">Krungthai Bank</string>
     <string name="payment_method_promptpay_title">PromptPay</string>
     <string name="payment_method_paynow_title">PayNow</string>
+    <string name="payment_truemoney_title">TrueMoney Wallet</string>
+    <string name="truemoney_info_text">Please input mobile number connected to your TrueMoney Wallet account</string>
```

Comment:
```suggestion
    <string name="truemoney_info_text">Please input the mobile number connected to your TrueMoney Wallet account</string>
```

Reply:
Thanks, here is the updated commit [428d6a5](https://github.com/omise/omise-android/pull/130/commits/428d6a5367505af43ac4097137935d1fed97c9dc).